sos/plugins/openvswitch.py

```diff
@@ -147,24 +235,81 @@ def setup(self):
                             "ovs-ofctl -O %s dump-ports-desc %s" % (flow, br)
                         ])
 
-        # Gather info on the DPDK mempools associated with each DPDK port
-        br_list_result = self.collect_cmd_output("ovs-vsctl -t 5 list-br")
-        if br_list_result['status'] == 0:
-            for br in br_list_result['output'].splitlines():
-                port_list_result = self.exec_cmd(
+                port_list_result = self.call_ext_prog(
                     "ovs-vsctl -t 5 list-ports %s" % br
                 )
                 if port_list_result['status'] == 0:
                     for port in port_list_result['output'].splitlines():
-                        self.add_cmd_output(
-                            "ovs-appctl netdev-dpdk/get-mempool-info %s" % port
-                        )
+                        self.add_cmd_output([
+                            "ovs-appctl cfm/show %s" % port,
+                            "ovs-appctl qos/show %s" % port,
+                            # Not all ports are "bond"s, but all "bond"s are
+                            # a single port
+                            "ovs-appctl bond/show %s" % port,
+                        ])
+
+                        if check_dpdk:
+                            self.add_cmd_output(
+                                "ovs-appctl netdev-dpdk/get-mempool-info %s" %
+                                port
+                            )
+
+                if check_dpdk:
+                    iface_list_result = self.call_ext_prog(
```

Comment:
Same here, do not try to use `call_ext_prog()` as it does not exist in the 3.9 API.